artifacthub/library/general/storageclass/1.0.1/artifacthub-pkg.yml

```diff
@@ -1,7 +1,7 @@
 version: 1.0.1
 name: k8sstorageclass
 displayName: Storage Class
-createdAt: "2022-11-08T23:45:23Z"
```

Comment:
why does this need to be updated?

Reply:
not sure, i just ran generate. reverted it now, it doesn't regen the timestamp.